### ISSUE TITLE
Rerun wiki names import on per-country repos

### ISSUE THREAD

**Issue by stepps00** [whosonfirst-data/whosonfirst-data#1656]
The last Wikidata/Wikipedia names import occurred three years ago - since then, various changes have occurred in Who's On First and millions of new records have been created.

@.okavvada crafted [tools](https://github.com/mapzen-data/wikipedia-notebooks) to automate the import of new names into Who's On First; let's re-run the import to add as many new name translations to the per-country repos as possible.

While we're at it, let's output metrics to figure out how many new names were added and what languages these new name translations are in.

**Comment by stepps00** [comment]
All repos have been updated with new wikidata names, aside from the Switzerland repository.

Once #1334 is complete, I'll run this same work against the admin data in Switzerland. I'll keep this issue open until that is complete.

**Comment by stepps00** [comment]
In total, this work has added an additional 1,248,113 **new** preferred name properties. This number does not include the additional variant properties that were added or appended to.

I'll update this issue with a total count of records updated. Here is a breakdown of language counts:

lang|count
--|--
aar|242
abk|112
ace|596
ady|66
aeb|1
afr|4775
aka|601
aln|233
als|205
amh|800
ang|636
anp|2
ara|9411
arc|189
arg|3829
arn|262
arq|4
ary|20
arz|960
asm|207
ast|24719
atj|520
ava|169
avk|170
aym|584
azb|34785
aze|4065
bak|1549
bam|1824
ban|180
bar|4574
bbc|170
bcl|866
bel|6964
ben|4880
bgn|78
bho|1219
bih|130
bis|369
bjn|245
bod|332
bos|1782
bpy|2504
bqi|3
bre|5534
brh|85
bto|170
bug|99
bul|6214
bxr|230
cat|12468
cdo|3411
ceb|151458
ces|10841
cha|315
che|1168
cho|241
chr|115
chu|113
chv|538
chy|336
ckb|771
cor|669
cos|3416
cps|170
cre|61
crh|1122
csb|395
cym|8296
dan|12746
deu|24078
din|249
diq|1083
div|78
dsb|550
dtp|188
dty|505
dzo|27
egl|178
ell|6280
eml|804
eng|18838
epo|10956
est|8111
eus|29346
ewe|296
ext|509
fao|2178
fas|37642
fij|347
fin|10111
fit|340
fra|24541
frc|1730
frp|2939
frr|2529
fry|1634
ful|281
fur|3530
gag|539
gan|1436
gcr|47
gla|3806
gle|6542
glg|6251
glk|41
glv|523
gom|94
gor|555
got|105
grc|33
grn|790
gsw|3686
guj|2726
hak|1324
hat|1216
hau|1142
haw|529
heb|8838
her|243
hif|893
hil|228
hin|4358
hmo|241
hrv|5737
hrx|175
hsb|1925
hun|11119
hye|5612
hyw|601
ibo|349
ido|3982
iii|3
iku|78
ile|3623
ilo|1367
ina|3801
ind|8939
inh|296
ipk|304
isl|4345
ita|28466
jam|1828
jav|1498
jbo|314
jpn|15939
jut|217
kaa|430
kab|1956
kal|1850
kan|2950
kas|28
kat|6481
kau|236
kaz|3143
kbd|88
kbp|662
kea|174
khm|127
kik|452
kin|379
kir|3594
kjp|17
koi|94
kom|148
kon|2895
kor|7644
krc|84
kri|217
krj|230
krl|170
ksh|319
kua|189
kur|711
lad|1560
lao|135
lat|2966
lav|5484
lbe|110
lez|203
lfn|1054
lij|3187
lim|3468
lin|476
lit|6658
liv|248
lmo|1405
loz|165
lrc|235
ltg|271
ltz|6923
lug|251
lus|165
lzh|1895
lzz|15
mah|227
mai|756
mal|2652
mar|2888
mdf|47
mhr|290
min|3306
mkd|3332
mlg|6196
mlt|486
mon|801
mri|479
mrj|643
msa|8347
mus|236
mwl|618
mya|746
myv|990
mzn|334
nah|1198
nan|27623
nap|3046
nau|471
nav|749
ndo|190
nds|5086
nep|719
new|762
niu|172
nld|23598
nno|7910
nob|14861
nor|2114
nov|437
nrm|3023
nso|465
nya|353
nys|187
oci|5197
olo|683
ori|484
orm|104
oss|832
ota|5
pag|547
pam|896
pan|901
pap|1964
pcd|2992
pdc|393
pdt|235
pfl|675
pih|482
pli|97
pms|3026
pnb|1772
pnt|59
pol|28749
por|14628
prg|1601
pus|499
que|1297
qug|185
rgn|1549
rif|184
rmy|174
roh|3041
ron|15581
rue|407
run|266
rup|817
ruq|47
rus|16158
sag|266
sah|480
san|423
sat|600
scn|3612
sco|7249
sdc|186
sei|166
sgs|760
shn|128
sin|2345
sje|181
sli|183
slk|6007
slv|5251
sma|293
sme|682
smj|189
smn|44
smo|342
sms|45
sna|483
snd|512
som|661
sot|353
spa|31928
sqi|4599
srd|3433
srn|297
srp|15721
srq|170
ssw|250
stq|631
sun|786
swa|4700
swe|23634
szl|2857
tah|609
tam|3374
tat|4532
tay|51
tcy|131
tel|3121
tet|319
tgk|1386
tgl|2531
tha|3720
tir|4
tly|2
ton|255
tpi|352
tru|13
tsn|284
tso|335
tuk|316
tum|293
tur|14290
twi|463
tyv|80
tzm|6
udm|358
uig|420
ukr|16291
urd|10805
uzb|2252
vec|4543
ven|280
vep|855
vie|12028
vls|3119
vmf|1639
vol|7355
vot|69
vro|599
war|7121
wln|3111
wol|2937
wuu|3083
xal|88
xho|363
xmf|1080
yid|790
yor|740
yue|4707
zea|539
zgh|2
zha|272
zho|24434
zul|2916

**Comment by nvkelso** [comment]
Most excellent!

I see the ‘ceb’ bot lives on, but so be it.

**Comment by stepps00** [comment]
The second round of name additions added roughly 10x the amount of names the first run added, breakdown of the second run below:

lang|count
--|--
abk|2451
ace|6199
ady|2859
afr|34084
aka|2044
als|21093
amh|13216
ang|6585
ara|106197
arc|3675
arg|31079
arz|11162
asm|2672
ast|41906
atj|606
ava|2204
aym|8174
azb|58497
aze|47876
bak|16676
bam|6649
bar|16362
bcl|9492
bel|60904
ben|35886
bih|17141
bis|2576
bjn|2555
bod|9676
bos|39004
bpy|37033
bre|62921
bug|9461
bul|82666
bxr|6023
cat|168720
cdo|14150
ceb|472675
ces|115940
cha|1975
che|26380
cho|35
chr|1823
chu|1630
chv|11482
chy|4369
ckb|14489
cor|11133
cos|7261
cre|355
crh|4517
csb|3347
cym|40687
dan|69804
deu|418256
din|154
diq|10226
div|2655
dsb|5776
dty|6060
dzo|655
ell|43174
eml|8885
epo|129363
est|64612
eus|139805
ewe|1898
ext|7148
fao|11806
fas|168888
fij|3850
fin|108957
fra|389304
frp|5057
frr|10932
fry|21354
ful|999
fur|3106
gag|4135
gan|4713
gla|17919
gle|26545
glg|64463
glk|882
glv|8064
gom|2099
gor|909
got|2050
grn|7906
guj|9805
hak|15149
hat|14007
hau|10308
haw|5728
heb|80563
hif|9629
hin|45212
hrv|61142
hsb|10982
hun|139350
hye|84419
hyw|3428
ibo|2043
ido|21507
iku|1531
ile|10743
ilo|26118
ina|23437
ind|84185
inh|3783
ipk|1265
isl|25152
ita|352977
jam|4258
jav|24640
jbo|3147
jpn|130426
kaa|5963
kab|7674
kal|4965
kan|14471
kas|620
kat|58452
kaz|68947
kbd|2310
kbp|5908
khm|2854
kik|12774
kin|3338
kir|24023
koi|2343
kom|3274
kon|4146
kor|99606
krc|3266
ksh|1523
kur|13000
lad|11959
lao|2099
lat|68099
lav|45728
lbe|1513
lez|3920
lfn|6504
lij|7252
lim|14721
lin|9312
lit|80791
lmo|34019
lrc|5501
ltg|1269
ltz|21489
lug|1969
mai|12064
mal|25069
mar|26956
mdf|1389
mhr|7095
min|5702
mkd|34524
mlg|74892
mlt|4074
mon|13491
mri|6741
mrj|10281
msa|160301
mwl|4489
mya|12294
myv|5859
mzn|9819
nah|10026
nap|14052
nau|5824
nav|3208
ndo|61
nds|54423
nep|20738
new|37009
nld|356516
nno|56574
nor|126069
nov|3616
nrm|3972
nso|1972
nya|2589
oci|61211
olo|4404
ori|11460
orm|1893
oss|15876
pag|3772
pam|18305
pan|22878
pap|7891
pcd|4767
pdc|4209
pfl|2575
pih|2784
pli|2374
pms|49474
pnb|32853
pnt|1383
pol|352618
por|261337
pus|9702
que|25543
rmy|4052
roh|5918
ron|143826
rue|5830
run|2562
rus|265588
sag|1309
sah|10218
san|10849
sat|2994
scn|24939
sco|83739
shn|718
sin|6642
slk|106628
slv|60942
sme|5891
smo|1831
sna|6162
snd|7705
som|10037
sot|1991
spa|275325
sqi|32122
srd|11786
srn|2025
srp|174303
ssw|1278
stq|8741
sun|9723
swa|32209
swe|421987
szl|17083
tah|3797
tam|35841
tat|27857
tcy|1052
tel|18136
tet|3383
tgk|14115
tgl|49383
tha|38100
tir|524
ton|444
tpi|2404
tsn|705
tso|1144
tuk|3984
tum|1669
tur|86615
twi|7651
tyv|1316
udm|4227
uig|12821
ukr|234899
urd|91166
uzb|65326
vec|15061
ven|414
vep|14571
vie|171534
vls|4642
vol|124627
war|136634
wln|4781
wol|3607
wuu|14436
xal|2812
xho|2044
xmf|13794
yid|14911
yor|11951
zea|5865
zha|4633
zho|143893
zul|3254

Total new (preferred) language properties added: 10,517,491
Total records updated: _incoming_

**Comment by stepps00** [comment]
To summarize...

First run:
- 1,248,113 new name properties added
- 176,834 records updated

Second run:
- 10,517,491 new name properties added
- 795,324 records updated

Total:
- 11,765,604 new name properties added
- 972,158 records updated

I will close this issue once all per-country PRs are linked and merged.

**Comment by nvkelso** [comment]
Fantastic!

**Comment by stepps00** [comment]
Since the large amount of linked PRs are hiding some comments..:

**First run:**
  - 1,248,113 new name properties added
  - 176,834 records updated

**Second run:**
  - 10,517,491 new name properties added
  - 795,324 records updated

**Total:**
  - 11,765,604 new name properties added
  - 972,158 records updated

There is a small amount of followup work to do in Switzerland to wrap this issue up. I'll file a new issue to track that work and close this issue, as the majority of name additions have been merged to master in their respective per-country repos.
